d3a_interface/constants_limits.py

```diff
@@ -102,11 +104,13 @@ class LoadSettings:
         # Max load energy rate, in ct/kWh
         FINAL_BUYING_RATE = 35
         FINAL_BUYING_RATE_RANGE = [0, 10000]
+        RATE_CHANGE_PER_UPDATE = [0, 100]
 
     class PVSettings:
         PANEL_COUNT_RANGE = [1, 10000]
         MIN_SELL_RATE_RANGE = [0, 10000]
         INITIAL_RATE_RANGE = [0, 10000]
+        RATE_CHANGE_PER_UPDATE = [0, 100]
```

Comment:
Why the limit to 100? I imagine, that the user wants to update the the price more that 10 Euros for each step, if the Final buying rate can be 10000. Also: Please make it consistent and use 10000 for all the rate ranges (sometimes you used 1000) Thanks!

Reply:
Personally, I think that all these limits are overly exaggerated. Still, there might be the case when the user might want to increase the price more than 1 Euro/kWh per update (100 cts/kWh -> 1 Euro/kWh). Therefore I agree with Hannes to harmonise this to the other rate parameters.